### DISCUSSION TITLE
Lag/delays with iterm2 3.5.0

### DISCUSSION THREAD

**Discussion by kochalex** [maxgoedjen/secretive#551] [General]
Not sure if anyone else is seeing this - but when updating iterm2 yesterday, I started getting 5 second or so delays when going ssh or git/ssh things

https://gitlab.com/gnachman/iterm2/-/issues/11483

**Comment by Tenzer** [comment]
I am seeing the same. I originally thought it was due to the macOS Sonoma 14.5 upgrade, but I can see there's a significant difference in speed with Git commit signing operations (using Secretive) between iTerm 3.5.0 and the stock Terminal.app.
